doc/2/controllers/bucket/create/index.md

```diff
@@ -0,0 +1,91 @@
+---
+code: true
+type: page
+title: create
+---
+
+# create
+
+Creates a S3 bucket
+
+## Query Syntax
+
+### HTTP
+
+```http
+URL: http://kuzzle:7512/_plugin/s3/bucket/create
+Method: POST
+Body:
+```
+
+```js
+      {
+        "bucketName":"<bucketname>",
+        "bucketRegion":"<bucketregion>",
+        "bucketOptions":{ <OptionsList> },
+        "bucketPolicy":{ <PolicyList> },
+        "bucketCORS":{ <CORS> }
+        "disableDotsInName":"false"
+      }
+```
+
+### Other protocols 
+
+```js
+{
+  "controller": "s3",
+  "action": "create",
+
+  "body": {
+      "bucketName": "<expression>",
+      "bucketRegion": "<number_of_result_max>",
+      "bucketOptions": { <OptionsList> },
+      "bucketPolicy":{ <PolicyList> },
+      "bucketCORS":{ <CORS> }
+      "disableDotsInName":"false"
+  }
```

Comment:
`bucketName`, `bucketRegion` and `disableDotsInName` (if `request.getBoolean` is used) are not in the body but they are top level arguments.

If you want body arguments, then you should use the `request.getBodyString` and `request.getBodyBoolean` helpers.